### PR TITLE
Document Urban Airship Integration

### PR DESCRIPTION
**Pull request by csexton**
This depends on the new release of Proximity Kit SDK.

I would appreciate a proof-read, because we all know I can't words that good.

/cc @cupakromer @syoder @mharper

### CODE REVIEW

File: docs/urban-airship.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Urban Airship Integration
+
+Proximity Kit offers first class integration with the Urban Airship SDK. Proximity Kit levereges our meta data attributes to allow the integration settings to be configured remotely.
+
+This integration combines Proximity Regions with Urban Airship's Tags and Events.
+
+Note: This functionality is currently iOS only. Android coming soon, if you want to be updated when it is avaliable [let us know](mailto:support@radiusnetworks.com).
+
```
Comment (mharper) [inline docs/urban-airship.md]: Android _support is_ coming soon.  If you want to be updated when it is **available** (typo)...

File: docs/urban-airship.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# Enable Integration in your App
+
+To enable integration in the SDK, you will need to set the Urban Airship manager on the proximity kit manager. This is a simple one-liner:
+
```
Comment (mharper) [inline docs/urban-airship.md]: s/proximity kit/Proximity Kit/

File: docs/urban-airship.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+[self.pkManager setAirship: [UAirship shared]];
+```
+
+That's it. Now the Proximity Kit SDK will look for the meta data attributes and handle the rest of the integration on the actual device. Now all you need to do is set tags or events.
+
+# Tags
+
+Urban Airship uses tags as a way to group different devices. Proximity Kit will add and remove tags as they enter and exit a region--for both Beacon Regions and Geofence Regions. In this way you can use the tag as an indication that the device is within a given region.
+
+To add and remove tags in Urban Airship, you will need to add a `ua_tags` meta data attribute to you region.
```
Comment (mharper) [inline docs/urban-airship.md]: s/you region/your region/

File: docs/urban-airship.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+This can be done by:
+
+- Edit the Region
+- Click on the "Add Meta Data Attribute"
+- Set the key to `ua_tags`
+- Set the value to a comma seperated list of tags
```
Comment (mharper) [inline docs/urban-airship.md]: s/comma seperated/comma-separated/

**Comment by mharper** [comment]
Apologies for commenting after the merge.

**Comment by csexton** [comment]
Better late than never, thanks.

**Pull request merged**